packages/design-system/src/components/Combobox/Combobox.tsx

```diff
@@ -400,6 +414,12 @@ const Content = styled(ComboboxPrimitive.Content)`
   }
 `;
 
+const CreateItemContainer = styled(Box)`
```

Comment:
The sticky item can't be reached with the keyboard. We need to fix this for the sake of accessibility.

Reply:
Are you sure it is not accessible? I can click on the button using the Up arrow

https://github.com/user-attachments/assets/df9cf0e7-b850-46b5-8440-243fb8af526a

Reply:
Yes but this isn't how it should work unfortunately. How can keyboard users understand that they have to press Up to focus the footer 😬. It is not accessible by pressing Down after the last option same with tab, we cannot reach the footer.

Reply:
Please check now